src/components/RichTextEditor/styling-controls/BoldButton.tsx

```diff
@@ -1,12 +1,13 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import React, { FC } from "react";
 import IconBold from "@foundation/Icon/Generated/IconBold";
 import { IconSize } from "@foundation/Icon/IconSize";
+import React, { FC } from "react";
 import { InlineStyleButton } from "../InlineStyleButton";
+import { Styles } from "../renderer/renderInlineStyles";
 
 export const BoldButton: FC = () => (
```

Comment:
IMHO it doesn't make much sense to create separate components for each button. You could use them directly in the toolbar since each button is only used once.

Reply:
c2a0b43